Dockerfile

```diff
@@ -104,6 +115,13 @@ FROM common AS runner
 
 ENV GODEBUG=cgocheck=0
 
+# copy watcher shared library
+COPY --from=builder /usr/local/lib/libwatcher* /usr/local/lib/
+# fix for the file watcher on arm
+RUN apt-get install -y --no-install-recommends gcc && \
```

Comment:
Shouldn't `libstdc++6` be enough? It can be installed in the same command as other packages btw.

Reply:
I don't remember exactly why, but the arm build fails if gcc is not installed here. I'll try building it again and see what the error message was

Reply:
Maybe could we add it only to the ARM build then? It's not very good to have a compiler available in the prod container from a security POV.

Reply:
Installing libstdc++6 was indeed enough 👍 . I guess apt-get and apk will automatically pull libgcc_s.so if it's not available yet.